pre-processing/primod/mapping/wel_svat_mapping.py

```diff
@@ -71,7 +73,7 @@ def _create_well_id(
         layer = np.tile(well_layer, (n_subunit, 1))
         layer_1d = layer[well_active]
 
-        well_id = self.well.dataset.coords["index"] + 1
+        well_id = well_cellid.coords["ncellid"] + 1
```

Comment:
ncellid is actually an index right? otherwise the +1 is now unnecessary.

Reply:
That is correct!